README.md

```diff
@@ -34,6 +34,9 @@ php occ calendar-resource:story:create 1 "2nd floor"
 php occ calendar-resource:room:create --wheelchair-accessible=1 --capacity=25 --room-number=201 1 "demouser" "berlin_main_office" "room.berlin.main@spacexyz.com" "Shared office"
 ```
 
+CAVEAT: each room needs an email different from any other room email -> (workaround: use fake email like "room0001@none" consecutively)
+ref https://github.com/nextcloud/calendar_resource_management/issues/119#issuecomment-2114275319
```

Comment:
```suggestion
CAVEAT: Each room needs a unique email address. A common workaround is to use fake email addresses like "room0001@none".
Ref https://github.com/nextcloud/calendar_resource_management/issues/119#issuecomment-2114275319
```

Let's use full sentences like the rest of the section.

Reply:
THX and looks good to me ...